src/mruby-zest/example/ZynPadSynth.qml

```diff
@@ -14,6 +14,7 @@ Widget {
         TabButton { label: "harmonic structure";}
         TabButton { label: "oscillator";}
         TabButton { label: "envelopes & lfos"; whenClick: lambda {root.set_view_pos(:vis, :envelope)};}
+        TabButton { whenClick: lambda {header.set_tab(3)}; label: "resonance"}
```

Comment:
I don't see why the whenClick handler is needed, the TabButton class already performs this action. I'll remove it in a cleanup commit as the rest of the PR appears to function as intended.